### PR TITLE
Admin Pages

### PR DESCRIPTION
**Pull request by tbwrigh**

### CODE REVIEW

File: backend/src/main.py [hunk 1]
```diff
@@ -9,6 +10,15 @@
 load_dotenv()
 
 app = FastAPI()
+
+app.add_middleware(
```
Comment (OxygenCobalt) [inline backend/src/main.py:14]: Can you explain why this was added? Sorry not too familiar with cors

File: frontend-admin/src/pages/routes/routes-page.tsx [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
 import Card from '../../components/card/card';
 import './routes-page.scss';
 
+interface Route {
+  id: number;
+  name: string;
+}
+
+const fetchRoutes = async () => {
+  const response = await fetch('http://localhost:8000/routes');
```
Comment (OxygenCobalt) [inline frontend-admin/src/pages/routes/routes-page.tsx:12]: `http://localhost:8000/routes/` (Convention, less likely to be rejected by any id parser)

Comment (OxygenCobalt) [inline frontend-admin/src/pages/routes/routes-page.tsx:12]: Eventually, we will want to dynamically switch between this dev server and a prod server. See the app for an example.

File: frontend-admin/src/pages/vans/vans-page.tsx [hunk 3]
```diff
@@ -1,15 +1,34 @@
...
+interface Van {
+  vanId: number;
+  routeId: number;
+  wheelchair: boolean;
+}
+
+const fetchVans = async () => {
+  const response = await fetch('http://localhost:8000/vans');
+  const data = await response.json();
+  const van_data = data["vans"] as Van[];
```
Comment (OxygenCobalt) [inline frontend-admin/src/pages/vans/vans-page.tsx:15]: Will this throw an error if it's not in the right structure?

**Review by OxygenCobalt — CHANGES_REQUESTED** [review]

File: backend/src/main.py (hunk 1 above)
Comment (tbwrigh) [reply backend/src/main.py:14]: need cors middleware to access backend from frontend, but this overly loose will tighten it in the future

File: frontend-admin/src/pages/routes/routes-page.tsx (hunk 2 above)
Comment (tbwrigh) [reply frontend-admin/src/pages/routes/routes-page.tsx:17]: yep, just hardcoding for demo

Comment (tbwrigh) [reply frontend-admin/src/pages/routes/routes-page.tsx:17]: yep, just hardcoding for demo

File: backend/src/main.py (hunk 1 above)
Comment (OxygenCobalt) [reply backend/src/main.py:14]: Ok

File: frontend-admin/src/pages/routes/routes-page.tsx (hunk 2 above)
Comment (tbwrigh) [reply frontend-admin/src/pages/routes/routes-page.tsx]: fine ig, for the routes where we end with the routeID should we also include a trailing slash?

File: frontend-admin/src/pages/vans/vans-page.tsx (hunk 3 above)
Comment (tbwrigh) [reply frontend-admin/src/pages/vans/vans-page.tsx:36]: unsure, just getting stuff done for the demo

File: frontend-admin/src/pages/routes/routes-page.tsx (hunk 2 above)
Comment (OxygenCobalt) [reply frontend-admin/src/pages/routes/routes-page.tsx]: Yes

**Review by ezrichards — APPROVED** [review]
We're probably good to do a preliminary merge if you want.

**Review by OxygenCobalt — APPROVED** [review]
LGTM.

**Pull request merged**